### PR TITLE
Use async / await syntax in seeds as default

### PR DESCRIPTION
**Pull request by vencho-mdp**

### CODE REVIEW

File: lib/migrations/seed/stub/js.stub [hunk 1]
```diff
@@ -2,15 +2,13 @@
...
-        {id: 3, colName: 'rowValue3'}
-      ]);
-    });
+  await knex('table_name').del()
+  await knex('table_name').insert([
+    {id: 1, colName: 'rowValue1'},
+    {id: 2, colName: 'rowValue2'},
+    {id: 3, colName: 'rowValue3'}
+  ]);
+  return knex
```
Comment (kibertoad) [inline lib/migrations/seed/stub/js.stub:13]: is returning knex even necessary here?

Comment (vencho-mdp) [reply lib/migrations/seed/stub/js.stub:13]: Good point, I don't think it is, but I'm not really sure

Comment (kibertoad) [reply lib/migrations/seed/stub/js.stub:13]: Original was just returning the promise from seed, so this should not be needed.

**Review by kibertoad — APPROVED** [review]

**Comment by kibertoad** [comment]
thank you!

**Comment by kibertoad** [comment]
Released in 1.0.3

**Pull request merged**